### PR TITLE
new cis card, new keys, scheme signed

### PR DESCRIPTION
**Pull request by yaelgoede**
Changed some attributes for the cis card. Also created new keys and signed the scheme.

### CODE REVIEW

File: cis/Issues/complianceCheck/description.xml [hunk 1]
```diff
@@ -80,6 +60,26 @@
...
+			</Description>
+		</Attribute>
+		<Attribute id="digidlevel" displayIndex="5">
+			<Name>
+				<en>Assurance level</en>
+				<nl>Betrouwbaarheidsniveau</nl>
+			</Name>
+			<Description>
+				<en>The assurance level with which your identity was verified</en>
+				<nl>Het betrouwbaarheidsniveau waarmee je identiteit is vastgesteld</nl>
```
Comment (sanderhollaar) [inline cis/Issues/complianceCheck/description.xml:80]: 'is' or 'was' ?

File: cis/Issues/complianceCheck/description.xml [hunk 2]
```diff
@@ -80,6 +60,26 @@
 				<nl>Je geslacht</nl>
 			</Description>
 		</Attribute>
+		<Attribute id="nationality" optional="true" displayIndex="4" displayHint="yesno">
+			<Name>
+				<en>Dutch nationality</en>
+				<nl>Nederlandse nationaliteit</nl>
+			</Name>
+			<Description>
+				<en>Whether you have the dutch nationality</en>
```
Comment (sanderhollaar) [inline cis/Issues/complianceCheck/description.xml:69]: Dutch

File: cis/Issues/complianceCheck/description.xml [hunk 3]
```diff
@@ -80,6 +60,26 @@
...
+		<Attribute id="digidlevel" displayIndex="5">
+			<Name>
+				<en>Assurance level</en>
+				<nl>Betrouwbaarheidsniveau</nl>
+			</Name>
+			<Description>
+				<en>The assurance level with which your identity was verified</en>
+				<nl>Het betrouwbaarheidsniveau waarmee je identiteit is vastgesteld</nl>
+			</Description>
+		</Attribute>
```
Comment (ivard) [inline cis/Issues/complianceCheck/description.xml:82]: It's currently not supported to remove existing attributes. The only possible way now is to add `optional="true"` to the attributes you want to get rid of, and then you can skip them on issuance. New attributes should be added to the end of the XML list. The visible order can be changed using `displayIndex`.

File: cis/Issues/complianceCheck/description.xml (hunk 1 above)
Comment (yaelgoede) [reply cis/Issues/complianceCheck/description.xml:80]: good point

**Review by sanderhollaar — DISMISSED** [review]

File: cis/Issues/complianceCheck/description.xml [hunk 4]
```diff
@@ -93,5 +93,25 @@
...
+		<Attribute id="digidlevel" displayIndex="5">
+			<Name>
+				<en>Assurance level</en>
+				<nl>Betrouwbaarheidsniveau</nl>
+			</Name>
+			<Description>
+				<en>The assurance level with which your identity was verified</en>
+				<nl>Het betrouwbaarheidsniveau waarmee je identiteit was vastgesteld</nl>
+			</Description>
+		</Attribute>
```
Comment (ivard) [inline cis/Issues/complianceCheck/description.xml:115]: `digidlevel` is a new attribute, so it should be optional.

**Review by sanderhollaar — APPROVED** [review]

**Pull request merged**